### PR TITLE
Patch 248 | Use of unsafe html templating. 

### PR DESCRIPTION
**Pull request by Tjoosten**
Convert `{!! !!}` to ´{{ }}`

### CODE REVIEW

File: resources/views/contributors/home.blade.php [hunk 1]
```diff
@@ -1,5 +1,5 @@
 <!DOCTYPE html>
-<html lang="{!! Config::get('app.locale');!!}" ng-app="irailapp" ng-controller="PlannerCtrl">
+<html lang="{{ Config::get('app.locale'); }}" ng-app="irailapp" ng-controller="PlannerCtrl">
```
Comment (Haroenv) [inline resources/views/contributors/home.blade.php]: Remove the semicolon here, it's one of the reasons the test fails

**Comment by pietercolpaert** [comment]
Thanks @Tjoosten! Can you process https://github.com/iRail/hyperRail/pull/251#discussion_r89584187 still? Then we can merge this PR

**Comment by Tjoosten** [comment]
Sure later on the day. I'm vurrently working on something else that i want to finish.

**Comment by pietercolpaert** [comment]
Thanks! Merged!

**Comment by Tjoosten** [comment]
No problem

**Pull request merged**